core/src/main/java/org/mapfish/print/map/image/ImageLayer.java

```diff
@@ -157,30 +169,47 @@ public double getImageBufferScaling() {
   public void prepareRender(
       final MapfishMapContext transformer,
       final MfClientHttpRequestFactory clientHttpRequestFactory) {
-    image = fetchImage(transformer, clientHttpRequestFactory);
+    try {
+      image = fetchImage(transformer, clientHttpRequestFactory);
+    } catch (Exception e) {
```

Comment:
please catch only the 2 exceptions from fetchImage and RuntimeException if required